wrappers/services.go

```diff
@@ -241,6 +242,8 @@ func AddSnapServices(s *snap.Info, disabledSvcs []string, inter interacter) (err
 		}
 	}()
 
+	preseedMode := release.PreseedMode
```

Comment:
this needs a TODO that we want to avoid this as soon as possible

Reply:
Added (there is also similiar TODO in the release package).